boards/arm/rddrone_fmuk66/rddrone_fmuk66-pinctrl.dtsi

```diff
@@ -53,7 +53,7 @@
 	ftm0_default: ftm0_default {
 		group0 {
 			pinmux = <FTM0_CH0_PTC1>,
-				<FTM0_CH1_PTA4>,
+				<FTM0_CH3_PTA6>,
```

Comment:
Is this change related to the regulator? It doesn't seem to belong in commit `boards:rddrone_fmuk66 Enable 3.3v regulator`